monai/transforms/utils.py

```diff
@@ -2051,5 +2058,136 @@ def has_status_keys(data: torch.Tensor, status_key: Any, default_message: str =
     return True, None
 
 
+def distance_transform_edt(
+    img: NdarrayOrTensor,
+    sampling: None | float | list[float] = None,
+    return_distances: bool = True,
+    return_indices: bool = False,
+    distances: NdarrayOrTensor | None = None,
+    indices: NdarrayOrTensor | None = None,
+    block_params: tuple[int, int, int] | None = None,
+    float64_distances: bool = False,
+) -> tuple[NdarrayOrTensor, NdarrayOrTensor]:
+    """
+    Euclidean distance transform, either GPU based with CuPy / cuCIM
+    or CPU based with scipy.
+    To use the GPU implementation, make sure cuCIM is available and that the data is a `torch.tensor` on a GPU device.
+
+    For details about the implementation, check out the
+    `SciPy<https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.distance_transform_edt.html/>`_
+    and `cuCIM<https://docs.rapids.ai/api/cucim/nightly/api/#cucim.core.operations.morphology.distance_transform_edt/>`_ docs.
+
+    Args:
+        img: Input image on which the distance transform shall be run.
+            channel first array, must have shape: (num_channels, H[, W, ..., ])
+            Input gets passed channel-wise to the distance-transform, thus results from this function may differ
+            from directly calling ``distance_transform_edt()`` in CuPy or scipy.
+        sampling: Spacing of elements along each dimension. If a sequence, must be of length equal to the input rank;
```

Comment:
Should this not be "length equal to input rank -1"? Since the first is the channel dimension?